atom/common/native_mate_converters/callback.h

```diff
@@ -72,8 +72,14 @@ struct V8FunctionInvoker<ReturnType(ArgTypes...)> {
     v8::Context::Scope context_scope(context);
     ReturnType ret;
     std::vector<v8::Local<v8::Value>> args = { ConvertToV8(isolate, raw)... };
-    v8::Local<v8::Value> val(holder->Call(holder, args.size(), &args.front()));
-    Converter<ReturnType>::FromV8(isolate, val, &ret);
+    v8::Local<v8::Value> result;
+    auto maybe_result =
+        holder->Call(context, holder, args.size(), &args.front());
+    if (!maybe_result.ToLocal(&result)) {
+      static ReturnType defaultType;
+      return defaultType;
```

Comment:
The mac build on travis was throwing `-Wsometime-uninitialised` when i did that https://travis-ci.org/atom/electron/jobs/87215154